network/p2p/p2p_sync.go

```diff
@@ -21,126 +18,8 @@ import (
 	"github.com/bloxapp/ssv/network/commons"
 	"github.com/bloxapp/ssv/protocol/v2/message"
 	p2pprotocol "github.com/bloxapp/ssv/protocol/v2/p2p"
-	"github.com/bloxapp/ssv/protocol/v2/ssv/queue"
 )
 
-func (n *p2pNetwork) SyncHighestDecided(mid spectypes.MessageID) error {
-	ctx := context.TODO() // TODO: pass context to SyncHighestDecided
-
-	return n.syncer.SyncHighestDecided(ctx, n.interfaceLogger, mid, func(msg *queue.DecodedSSVMessage) {
-		n.msgRouter.Route(ctx, msg)
-	})
-}
-
-func (n *p2pNetwork) SyncDecidedByRange(mid spectypes.MessageID, from, to qbft.Height) {
-	ctx := context.TODO() // TODO: pass context to SyncDecidedByRange
```

Comment:
I thought we said we wanted to leave sync for fullnodes and not just removed entirely. wouldn't just deleting like this stop fullnode/explorer functionality?

Reply:
Yes, but since the history sync was disabled anyway, and we want to redesign the syncing, then we preferred to fully align to spec here by removing `SyncDecidedByRange` as well.